css/buttons.css

```diff
@@ -0,0 +1,90 @@
+/* GENERAL */
+
+.button-div {
```

Comment:
`div-download` or `button-download-wrapper`

Reply:
I use it for the toggle as well though, so `download` isn't semantically correct. Would `button-wrapper` work?